src/userinfo/ioUserInfoClient.ts

```diff
@@ -0,0 +1,72 @@
+import * as TE from "fp-ts/lib/TaskEither";
+import * as f from "fp-ts/function";
+import nodeFetch from "node-fetch";
+import * as authClient from "../generated/clients/io-auth/client";
+import { UserIdentity } from "../generated/clients/io-auth/UserIdentity";
+import { Client, createClient } from "../generated/clients/io-auth/client";
+import * as I from ".";
+
+const toError = (type: I.ErrorType): I.UserInfoClientError => ({
```

Comment:
What do you think to move this function on `<pr-home>/src/userinfo/index.ts` file?

Reply:
Agree! Done!